enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/iverksett/SimuleringDto.kt

```diff
@@ -0,0 +1,27 @@
+package no.nav.familie.kontrakter.ef.iverksett
+
+import no.nav.familie.kontrakter.ef.felles.StønadType
+import java.time.LocalDate
+import java.util.UUID
+
+data class SimuleringDto(
+        val nyTilkjentYtelseMedMetaData: TilkjentYtelseMedMetadata,
+        val forrigeBehandlingId: UUID?
+)
+
+data class TilkjentYtelseMedMetadata(
+        val tilkjentYtelse: SimuleringTilkjentYtelseDto,
+        val saksbehandlerId: String,
+        val eksternBehandlingId: Long,
+        val stønadstype: StønadType,
+        val eksternFagsakId: Long,
+        val personIdent: String,
+        val behandlingId: UUID,
+        val vedtaksdato: LocalDate
+)
+
+data class SimuleringTilkjentYtelseDto(
+        val id: UUID = UUID.randomUUID(),
+        val vedtaksdato: LocalDate? = null,
```

Comment:
Vet ikke om vi trenger id og vedtaksdato her, vedtaksdato finnes ikke i iverksett sin dto for dette? 
Og ID trenger vi nog ikke, hvis den trengs burde den nok ikke settes til default UUID.randomUUID()

Reply:
Vedtaksdato finnes en nivå opp (TilkjentYtelseMedMetadata)

Reply:
Jeg fjerner vedtaksdato siden den allerede finnes i TilkjentYtelseMedMetadata

Reply:
Tar i bruk tilkjentYtelseDto istedenfor SimuleringTilkjentYtelseDto